lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -798,7 +840,11 @@ instance FromJSON SomeByronWalletPostData where
     parseJSON = withObject "SomeByronWallet" $ \obj -> do
         obj .: "style" >>= \case
             t | t == toText Random ->
-                SomeRandomWallet <$> parseJSON (Aeson.Object obj)
+                (obj .:? "passphrase_hash" :: Aeson.Parser (Maybe Text)) >>= \case
```

Comment:
The swagger schema says `"passphrase_hash"` is a required field, right?

Reply:
It is indeed.